Bottom: e0b8db38756517cb37063d235b13659a02487154
Top:    7c9c4ae215e2f1b4f5b6d58f09fbc268779eb525
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:48:58 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 68ec2e7..03d212a 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1254,8 +1254,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
